reflection/src/main/java/com/alevel_hw2/ContactDAO.java

```diff
@@ -0,0 +1,117 @@
+
+    private final String DELETE_CONTACT = "DELETE FROM contacts WHERE id = ?";
+
+    private final String READ_CONTACT = "SELECT * FROM contacts WHERE id = ?";
+
+    private final String READ_ALL_CONTACT = "SELECT * FROM contacts";
+
+    private Connection getConnection() {
+        try {
+            Connection connection = DriverManager.getConnection("jdbc:postgresql://localhost:5432/alevel",
+                    "postgres", "postgres");
+            return connection;
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public void createTable() {
+        try (Connection connection = getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(CREATE_TABLE_QUERY);
+            preparedStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public void createContact(String contact, String contact_type) {
+        try (Connection connection = getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(CREATE_CONTACT);
+
+            preparedStatement.setString(1, contact);
+            preparedStatement.setString(2, contact_type);
+
+            preparedStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public void readContact(long id) {
+        try (Connection connection = getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(READ_CONTACT);
+            preparedStatement.setLong(1, id);
+            try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                if (resultSet.next()) {
+                    int idContact = resultSet.getInt("id");
+                    String contact = resultSet.getString("contact");
+                    String contact_type = resultSet.getString("contact_type");
+                    System.out.println("Contact ID: " + idContact + ", Contact: " + contact + ", Contact_type: "
+                            + contact_type);
+                }
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public void readAllContact() {
+        try (Connection connection = getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(READ_ALL_CONTACT);
+            try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                while (resultSet.next()) {
+                    int idContact = resultSet.getInt("id");
+                    String contact = resultSet.getString("contact");
+                    String contact_type = resultSet.getString("contact_type");
+                    System.out.println("Contact ID: " + idContact + ", Contact: " + contact + ", Contact_type: " + contact_type);
```

Comment:
Here, it would be better to return List<Contact> (do relation mapping).
Then, you can just call this method, get a list and print it.